log/src/main/resources/db/migration/V1__Initial_version.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS usuario (
+    id BIGSERIAL PRIMARY KEY,
+    nome VARCHAR(100) NOT NULL,
+    profile VARCHAR(10) NOT NULL,
```

Comment:
```suggestion
    perfil VARCHAR(10) NOT NULL,
```
Não tinha percebido na modelagem do banco, mas aqui seria legal que fosse `perfil` ao invés de `profile` para seguir a linha de escrever em português.